src/main/java/interface_adapter/add/AddState.java

```diff
@@ -0,0 +1,45 @@
+package interface_adapter.add;
+
+import org.jgrapht.ListenableGraph;
+import org.jgrapht.graph.DefaultEdge;
+
+import java.util.List;
+
+public class AddState {
+    private ListenableGraph<String, DefaultEdge> graphT;
+    private String addError;
+
+    public AddState(ListenableGraph<String, DefaultEdge> graph) {
+        this.graphT = graph;
+    }
+
+    public ListenableGraph<String, DefaultEdge> getGraphT() {
+        return graphT;
+    }
+
+    public void setGraphT(ListenableGraph<String, DefaultEdge> graphT) {
+        this.graphT = graphT;
+    }
+
+    /**
```

Comment:
This does make more sense matching the name of the Use Case but just wondering if it should match the ViewModel